sleap_io/model/video.py

```
@@ -223,7 +226,28 @@
+        file_is_accessible = is_file_accessible(self.filename)
+        if not file_is_accessible:
+            return False
+
+        if dataset is None or dataset == "":
+            dataset = self.backend_metadata.get("dataset", None)
+
+        if dataset is not None and dataset != "":
+            has_dataset = False
+            if (
+                self.backend is not None
+                and type(self.backend) == HDF5Video
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)